__tests__/app.test.js

```diff
@@ -38,4 +38,29 @@ describe("app", () => {
       })
     });
   });
+
+  describe("GET /api", () => {
+    test("Responds with a status of 200 for the right request.", () => {
+      return request(app).get("/api").expect(200);
+    });
+    test("200 - Responds with an array of topics to the client.", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(typeof(body.endpoint)).toBe('object');
+          // // Checking the data type
+          for (const [key, value] of Object.entries(body.endpoint)) {
+            expect(typeof(`${key}`)).toBe('string');
```

Comment:
you could be more specific here because you know the keys that every object will have in your endpoints.json. An easier way to check that it has everything you expect is to require in the endpoints file into the test suite and check what comes back from the api matches that